app/assets/javascripts/active_admin/lib/batch_actions.es6

```diff
@@ -3,7 +3,7 @@ const onDOMReady = function() {
   // Use ActiveAdmin.modal_dialog to prompt user if
   // confirmation is required for current Batch Action
   //
-  $('.batch_actions_selector li a').click(function(event){
+  $('.batch_actions_selector li a').off('click').on('click', function(event){
```

Comment:
@sgara while you are testing this, could you try rewriting this in the delegated event handling style to see if it fixes the behavior in Turbolinks but also does not affect non-Turbolinks usage?

```js
$(document).on('click', '.batch_actions_selector li a', function(event){
```

It really should be using that style like we do elsewhere in this file. At least I'd prefer that it do that rather than the off/on switch here. I'm just not sure if that will ultimately resolve the Turbolinks issue.

Reply:
I wasn’t fond of off/on either 😅. I’ll give it a try

Reply:
Tested delegated style but couldn't get it working at all, don't really know why though. I moved `off` on its own to clear both click and complete handlers, and I figured this would be much easier to read.